### DISCUSSION TITLE
Where can I set the '--enable-preview'

### DISCUSSION THREAD

**Discussion by JohnWickzzz** [nvim-java/nvim-java#95] [Q&A]
Hi, I use some jdk preview future in my java project, when I start the project, shows error like
```
错误: 加载主类 xx.Application 时出现 LinkageError
        java.lang.UnsupportedClassVersionError: Preview features are not enabled
 for xx/Application (class file version 65.65535). Try running 
with '--enable-preview'
```
Where I should set the args?

**Comment by s1n7ax** [comment]
@buptlibo See methods here helps https://github.com/neoclide/coc-java/issues/131

**Reply by JohnWickzzz** [reply]
Sorry for reply later, I've been busy with work lately. And I tried a lot of things.
Like update the pom.xml maven plugin:
```
<configuration>
   <source>21</source>
   <target>21</target>
   <release>21</release>
   <compilerArgs>
      <arg>--enable-preview</arg>
    </compilerArgs>
</configuration>
```
add the follow code in .settings-ort.eclipse.jdg.core.prefs
```
org.eclipse.jdt.core.compiler.problem.enablePreviewFeatures=enabled
org.eclipse.jdt.core.compiler.problem.reportPreviewFeatures=ignore
```
add '--enable-preview' in .mvn/jvm.config

It's not always in effect for me, sometimes I still have this problem. But I think that 'config message' may help others or maybe will work in other java project. So I've summarized my efforts here.
At last, try 'mvn clean' and 'mvn install', It might have some effect.
